cstar/roms/input_dataset.py

```diff
@@ -432,6 +432,29 @@ def _update_partitioning_attribute(
             path: path.stat() for path in parted_files
         }
 
+    @property
+    def path_for_roms(self) -> list[Path]:
```

Comment:
Do you always want to keep a "complete list?" Would a regex here be more compact and capture what you need?